shell/pom.xml

```diff
@@ -67,6 +67,11 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-compress</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-client</artifactId>
```

Comment:
@maobaolong we have a shaded-hadoop dependency. Basically, we try to let users use whatever hadoop version they configured/used.
Because there are multiple problems related to using one version of hadoop dependency and also the shaded-hadoop dependency, we try not to add those Hadoop dependencies in modules other than ufs/hadoop.

@maobaolong can you try to remove this dependency but also fulfill the function that you want to add?